lua/plugins/ui.lua

```diff
@@ -52,7 +52,7 @@ return {
     init = function() require("astronvim.utils").load_plugin_with_func("nvim-notify", vim, "notify") end,
     opts = {
       on_open = function(win)
-        vim.api.nvim_win_set_config(win, { zindex = 1000 })
+        vim.api.nvim_win_set_config(win, { zindex = 150 })
```

Comment:
```suggestion
        vim.api.nvim_win_set_config(win, { zindex = 175 })
```

Ok, after some testing and playing around, I think 175 is a good default here. I checked the zindex of our default plugins and this seems to be a good default. If other people want different results then they should override this themselves.